packages/components/psammead-headings/src/index.jsx

```diff
@@ -10,12 +10,16 @@ import {
   GEL_FF_REITH_SANS,
   GEL_FF_REITH_SERIF,
 } from '@bbc/gel-foundations/typography';
+import { MEDIA_QUERY_TYPOGRAPHY } from '@bbc/gel-foundations/breakpoints';
 
 export const Headline = styled.h1`
   color: ${C_SHADOW};
   font-family: ${GEL_FF_REITH_SERIF};
   margin: 0; /* Reset */
-  padding: ${GEL_SPACING_QUAD} 0 ${GEL_SPACING_DBL} 0;
+  padding: ${GEL_SPACING_QUAD} 0 ${GEL_SPACING_QUAD} 0;
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_AND_LARGER} {
```

Comment:
The Zeplin has it use 32px at 375px, however this doesnt do it until 318px, likely need to use the next breakpoint up

Reply:
Hmmm `LAPTOP_AND_LARGER` is `min-width: 600px`, so is the right one to use in this case - will update 👍 .

But the name is confusing - 600px wide is smaller than your typical _tablet_, so `LAPTOP_AND_LARGER` makes no sense 🤔